tests/core/actions/test_forms.py

```diff
@@ -90,6 +90,80 @@ async def test_activate_with_prefilled_slot():
     ]
 
 
+async def test_switch_forms_with_same_slot():
+    """Tests switching of forms, where the first slot is the same in both forms.
+
+    Tests the fix for issue 7710"""
+
+    # Define two forms in the domain, with same first slot
+    slot_a = "my_slot_a"
+
+    form_1 = "my_form_1"
+    text_1a = f"Please provide the value for {slot_a} of form 1"
```

Comment:
what do you think of naming the `text_xa` items to this?
```suggestion
    utter_ask_form_1 = f"Please provide the value for {slot_a} of form 1"
```